ui/.eslintrc.js

```diff
@@ -24,71 +25,73 @@ module.exports = {
     }
   },
 
-  // These rules were ported over from our common-ui/tslint.json file
+  // Some of these rules were ported over from our common-ui/tslint.json file
   // The rules found here were ported and categorized using the tslint migration roadmap:
   // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md
 
   // The eslint rules can be found here: https://eslint.org/docs/rules/
   rules: {
-    // 'prettier/prettier': 'warn', // Possibly use prettier to handle some formatting
-
-    /* Best Practices */
-    // 'no-multi-spaces': 'warn',
-
-    /* Typescript specific */
-    '@typescript-eslint/explicit-member-accessibility': 'off',
-    // '@typescript-eslint/member-ordering': ['warn', { 'classExpressions': ['method', 'field'] }],
-    '@typescript-eslint/no-empty-interface': 'warn',
-    '@typescript-eslint/no-inferrable-types': ['warn', {ignoreParameters: true}],
-    '@typescript-eslint/no-non-null-assertion': 'warn',
-    // 'prefer-arrow/prefer-arrow-functions': ['warn'], // Lots of 'newable' functions in the code base
-    '@typescript-eslint/type-annotation-spacing': 'warn',
-    '@typescript-eslint/unified-signatures': 'warn',
-
-    /* Style */
-    '@typescript-eslint/prefer-function-type': 'warn',
-    // 'spaced-comment': 'warn',
-    '@typescript-eslint/consistent-type-definitions': ['warn', 'interface'],
-    'no-trailing-spaces': 'warn',
-    'no-undef-init': 'warn',
-    // 'brace-style': ['warn', '1tbs'],
-    // 'simple-import-sort/sort': 'warn',
-    'quotes': ['warn', 'single'],
-    // '@typescript-eslint/semi': 'warn',
-    // 'space-before-function-paren': ['warn', { 'anonymous': 'never', 'named': 'never', 'asyncArrow': 'always' }],
+    /* Code Quality */
 
-    /* Functionality */
+    'constructor-super': 'warn',
     'curly': 'warn',
+    'eqeqeq': ['warn', 'always', {'null': 'ignore'}],
     'guard-for-in': 'warn',
-    'no-restricted-imports': ['error', {paths: ['rxjs'], patterns: ['../']}],
-    'no-caller': 'warn',
     'no-bitwise': 'warn',
-    // 'no-console': 'warn',  // 69 instances as of 3 Jan 2022
-    'no-new-wrappers': 'warn',
+    'no-caller': 'warn',
     'no-debugger': 'warn',
-    'constructor-super': 'warn',
     'no-empty': 'warn',
     'no-eval': 'warn',
+    'no-fallthrough': 'warn', // For switch statements
     'no-irregular-whitespace': 'warn',
```

Comment:
nit: probably belongs under style

Reply:
Based on https://eslint.org/docs/rules/no-irregular-whitespace there doesn't appear to be any valid use for these characters, so that puts it in the "correctness" category for me